core/embed/rust/src/ui/display/loader/circular_thin.rs

```diff
@@ -0,0 +1,335 @@
+use crate::ui::{
+    constant,
+    constant::{screen, LOADER_INNER, LOADER_OUTER},
+    display,
+    display::{toif::Icon, Color},
+    geometry::{Offset, Point, Rect},
+};
+
+#[cfg(feature = "dma2d")]
```

Comment:
you can drop all gates on `dma2d` if this file is only included with dma2d enabled

Reply:
9b181470c51a8e07565e5474d9730546c064be98